dist/images/ovnkube.sh

```diff
@@ -501,8 +495,8 @@ ovs-server () {
   while true; do
     if /usr/share/openvswitch/scripts/ovs-ctl status &>/dev/null; then
       echo "warning: Another process is currently managing OVS, waiting 10s ..." 2>&1
-      sleep 10 & wait
```

Comment:
That's weird... did you look through the history to see why it was that way? I wonder if there's some subtlety like in the case where kubelet kills the pod during the `sleep`.

Reply:
@danwinship didn't find any history on this. I have added the wait back. Like you say, it helps in receiving signal whilst the script is sleeping in the background. Thanks.